src/nodes/delta/marginals.jl

```diff
@@ -0,0 +1,11 @@
+export DeltaMarginal
+
+struct DeltaMarginal
```

Comment:
Can't we use the regular `Marginal(dist)` structure here?

Reply:
This puzzles me as well, why would a marginal on a delta-node edge be any different?